app/views/publishers/home.html.slim

```diff
@@ -108,6 +108,9 @@ script type="text/html" id="confirm_default_currency_modal_wrapper"
             .dashboard-panel--content#uphold_connect
               .status-description
                 = uphold_status_description(current_publisher).html_safe
+              = link_to(uphold_authorization_description(current_publisher),
+                        "https://uphold.com/dashboard",
```

Comment:
I think we should use the `uphold_dashboard_url` from the secrets file so in dev we can click it and go to the staging uphold site